gamatrix-gog.py

```diff
@@ -103,7 +130,102 @@ def init_opts():
     }
 
 
-def build_config(args):
+def build_config(args: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I understand generally how the `typing` stuff works, but what does `Any` mean here?